packages/individual-packages/gen.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
can we convert this into a ts file so we get the usual typescript goodness.

Reply:
Done.

Reply:
Done.